.changeset/stale-donkeys-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@neo4j-cypher/language-support': patch
+---
+
+Fix incorrect completions when identifier overlaps with keyword
```

Comment:
Only comment is this should not be included in the pr I think and we should bunch them up before doing a release? Rather than creating this patch.

Reply:
This will not trigger a release, only add a changeset right? It's nice to add the changeset relating to the commit at the same time we do the commit IMO